src/renderer/pages/Basket/model/basket-page-model.ts

```diff
@@ -0,0 +1,68 @@
+import { combine, createEvent, createStore, restore, sample } from 'effector';
+
+import { walletModel } from '@entities/wallet';
+import { basketModel } from '@entities/basket';
+import { includes } from '@shared/lib/utils';
+import { BasketTransaction } from '@shared/core';
+import { getTransactionTitle } from '../../Operations/common/utils';
+
+const queryChanged = createEvent<string>();
+const txSelected = createEvent<BasketTransaction['id']>();
```

Comment:
`BasketTransaction['id']` is more dynamic but don't think we really need it, `ID` is enough
```suggestion
const txSelected = createEvent<ID>();
```